blog/tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+"""Test for views."""
+
+from datetime import datetime
+
+from django.test import TestCase, Client
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.urls import reverse
+
+from ..models import Post
+
+
+class PostViewTest(TestCase):
+    """Test for post views."""
+
+    USERNAME = 'testuser'
+    PASSWORD = 'password'
+
+    def setUp(self):
+        """Init test data."""
+        self.client = Client()
+        self.user = User.objects.create(username=self.USERNAME,
+                                        email='testuser@test.com',
+                                        is_superuser=True,
+                                        is_staff=True,
+                                        is_active=True)
+        self.user.set_password(self.PASSWORD)
```

Comment:
look for `Model.objects.create_superuser`, it should be possible to do all steps with one call.